lib/classes/Variables.test.js

```diff
@@ -990,6 +1024,11 @@ module.exports = {
   });
 
   describe('#overwrite()', () => {
+    beforeEach(() => {
```

Comment:
`this.variableSyntax` requires explicit loading but was previously not used in overwrite.  Therefore, loading it for successful tests is newly necessary